keyboards/ocean/stealth/rules.mk

```
@@ -0,0 +1,22 @@
+# MCU name
+MCU = atmega32u4
+
+# Bootloader selection
+BOOTLOADER = caterina
+
+# Build Options
+#   change yes to no to disable
+#
+BOOTMAGIC_ENABLE = no       # Virtual DIP switch configuration
```

We do highly recommend keeping bootmagic enabled. 
```suggestion
BOOTMAGIC_ENABLE = lite       # Virtual DIP switch configuration
```

---

i always forget about this, thanks!